docs/error-reporting/advanced/coroner/server-setup.md

```diff
@@ -0,0 +1,101 @@
+
+   - **SSL**: To change your SSL certificates, enter your key, certificate, and chain files in the provided fields.
+     <img src={useBaseUrl('/img/error-reporting/coroner/5e601ca786048.png')} alt="SSL" />
+
+5. **Configure Project**
+   When you select a project in the Projects section of the Organization config or select **Configure Project** from the upper-right menu while viewing a project in the dashboard, you access the project configuration page.
+
+   - **Tokens**: Submission tokens allow the upload of crash data and other symbols into a project. To create a token, click **Create a New Token** and then **Create**.
+     <img src={useBaseUrl('/img/error-reporting/coroner/5e601ca8739d6.png')} alt="tokens" />
+
+   - **Attributes**: Attributes define key metadata points used to generate indices. These coroner indices are used for filtering, visualization, and snapshot retrieval.
+
+     Creating an attribute for each metadata point added to your snapshots is recommended. For example, if you add "customer" to each snapshot belonging to this project, you should create an attribute named `customer`.
+
+     To create a new attribute for this project, click **Create a New Attribute** at the top of the Attributes section, and provide the attribute's name, type, format, and description. Refer to the Attributes Details section below for more information.
+
+     <img src={useBaseUrl('/img/error-reporting/coroner/5e601ca9c0f96.png')} alt="attributes" />
+
+   - **Attributes Details**:
+
+     - **Name**: Name of the attribute.
+     - **Type**: Data type of the attribute. The following types are supported:
+       - `dictionary`: Used to store low cardinality strings (less than 2 billion unique values). Examples include commits, versions, environment, etc. This encoding is efficient in that any string already present in the dictionary only requires 4 bytes of space to store in the column.
+       - `uint64`, `uint32`, `uint16`, `uint8`: Unsigned integer types used to store 64-bit, 32-bit, 16-bit, and 8-bit integers, respectively.
+     - **Format**: Format of the data type used for rendering the value. The following formats are supported:
+       - `kilobytes`, `gigabytes`, `terabytes`, `bytes`: The underlying column stores integer values treated as the specified unit. This is only valid for integer types.
+       - `callstack`: The underlying column stores a serialized JSON object representing a callstack as a sequence of strings, where each element represents a function. This is only valid for the dictionary type.
+       - `hostname`: The underlying column stores a machine hostname. This is only valid for the dictionary type.
+       - `memory_address`: The values represent memory addresses and should be rendered accordingly. This is only valid for integer types.
+       - `seconds`: The values represent seconds since a user-defined epoch (e.g., process.age is seconds since process start). This is only valid for integers.
```

Comment:
```suggestion
       - `seconds`: The values represent seconds since a user-defined epoch (for example., `process.age` is seconds since process start). This is only valid for integers.
```